source/scripting_v3/GTA/DirectionType.cs

```diff
@@ -0,0 +1,14 @@
+//
+// Copyright (C) 2023 kagikn & contributors
+// License: https://github.com/scripthookvdotnet/scripthookvdotnet#license
+//
+
+namespace GTA
+{
+	public enum DirectionType
```

Comment:
How about DirectionEnum?

Reply:
With syntax highlighting, we don't really need to add the suffix "Enum" to enum types. [Microsoft don't recommend that, either](https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-classes-structs-and-interfaces). Well, I often add `Flags` to enum types, but that is because I make enums identitial to enums defined in .sch files (the leaked ones).